client/src/components/mapping/Comments/index.tsx

```diff
@@ -22,78 +24,95 @@ const Comments = () => {
   const { attribute, resource } = useSelector(
     (state: IReduxStore) => state.selectedNode
   );
+  const me = useSelector((state: IReduxStore) => state.user);
 
   const attributesForResource = useSelector(
     (state: IReduxStore) => state.resourceInputs.attributesMap
   );
   const attributeForNode = attributesForResource[attribute.path];
-
+  console.log(attributeForNode);
   const [createAttribute] = useMutation(mCreateAttribute);
-  const [updateComments] = useMutation(mUpdateComments);
+  const [createComment] = useMutation(mCreateComment);
 
-  const [comments, setComments] = React.useState(
-    attributeForNode ? attributeForNode.comments : ''
-  );
+  const [newComment, setNewComments] = React.useState('');
+  const [comments, setComments] = React.useState([] as IComment[]);
 
-  const { data, loading } = useQuery(qCommentsForAttribute, {
+  const { data: attrWithComments, loading } = useQuery(qCommentsForAttribute, {
     variables: {
       attributeId: attributeForNode ? attributeForNode.id : null
     },
     skip: !attributeForNode
   });
 
   React.useEffect(() => {
-    setComments(data && data.attribute.comments ? data.attribute.comments : '');
-  }, [data]);
+    if (attrWithComments)
+      setComments(attrWithComments.attribute.comments.reverse());
```

Comment:
Should we sort by creation time instead?
Not sure but maybe the order of comments could be shuffled on import/export, deletion (when this will be supported), etc.

Reply:
ok that sounds good

Reply:
done